cns/configuration/configuration.go

```diff
@@ -107,6 +108,12 @@ type KeyVaultSettings struct {
 	RefreshIntervalInHrs int
 }
 
+type GRPCSettings struct {
+	EnableGRPC 					 bool
+	GRPCServerIPAddress  string
+	GRPCServerPort 			 uint16
```

Comment:
Looks like you might need to set up a fmter. `gofumpt`, I believe, is our preferred one.

Reply:
I have set up the `gofumpt` fmter thank you!